### PR TITLE
polish elasticdl local doc

### PR DESCRIPTION
**Pull request by QiJune**
For a better review, please refer to [here](https://github.com/QiJune/elasticdl/blob/feature/polish_doc/docs/tutorials/elasticdl_local.md)

### CODE REVIEW

File: docs/tutorials/elasticdl_local.md [hunk 1]
```diff
@@ -12,37 +12,43 @@ In this tutorial, we use [hyperkit](https://github.com/moby/hyperkit) as the hyp
...
 kubectl apply -f elasticdl/manifests/examples/elasticdl-rbac.yaml
 eval $(minikube docker-env)
+export DOCKER_BUILDKIT=1
 bash elasticdl/docker/build_all.sh
 ```
 
 ### Summit a training job
 
-There are other docker settings that you might also want to configure prior to submitting the training job. Please run `minikube docker-env` to get docker host url and docker cert path.
+There are other docker settings to configure prior to submitting the training job. Please run `minikube docker-env` to get docker host url and docker cert path.
```
Comment (skydoorkai) [inline docs/tutorials/elasticdl_local.md]: There are other docker settings to configure prior to submitting the training job.

For example:

```bash
export DOCKER_BASE_URL=tcp://192.168.64.5:2376
export DOCKER_TLSCERT=${HOME}/.minikube/certs/cert.pem
export DOCKER_TLSKEY=${HOME}/.minikube/certs/key.pem
```
We can get these setting values by running `minikube docker-env`.

File: docs/tutorials/elasticdl_local.md [hunk 2]
```diff
@@ -54,66 +60,101 @@ elasticdl train \
...
-  --output=model_output
-```
+We use the model predefined in model zoo directory. The model definition will be packed into the new docker image. The training and validation data are packaged to the base docker image already. We could use them directly.
+
+In this example, we use parameter server strategy. We launch a master pod, a parameter server(PS) pod and a worker pod. The worker pod gets model parameters from the PS pod, computes gradients and sends computed gradients to the PS pod. The PS pod iteratively updates these model parameters using gradients sent by the worker pod. For more details about parameter server strategy, please refer to the [design doc](https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/parameter_server.md).
 
 ### Check job status
 
-After submitting the job to Minikube, you can run following command to check the status of each pod:
+After submitting the job to Minikube, we can run following command to check the status of each pod:
```
Comment (skydoorkai) [inline docs/tutorials/elasticdl_local.md]: run following command -> run the following command

File: docs/tutorials/elasticdl_local.md [hunk 3]
```diff
@@ -54,66 +60,101 @@ elasticdl train \
...
 
 ```bash
 kubectl get pods
 ```
 
-You should see information on each pod like the following:
+We will see information on each pod like the following:
```
Comment (skydoorkai) [inline docs/tutorials/elasticdl_local.md]: We will see the information on each pod such as:

File: docs/tutorials/elasticdl_local.md [hunk 4]
```diff
@@ -54,66 +60,101 @@ elasticdl train \
...
+[2020-04-14 02:46:32,925] [INFO] [worker.py:879:_process_minibatch] Loss is 1.0977835655212402
+[2020-04-14 02:46:33,123] [INFO] [worker.py:879:_process_minibatch] Loss is 0.8362151384353638
+[2020-04-14 02:46:33,315] [INFO] [worker.py:879:_process_minibatch] Loss is 1.146580696105957
+[2020-04-14 02:46:33,518] [INFO] [worker.py:879:_process_minibatch] Loss is 1.4624073505401611
+[2020-04-14 02:46:33,648] [INFO] [worker.py:879:_process_minibatch] Loss is 0.9980261921882629
+[2020-04-14 02:46:33,851] [INFO] [worker.py:879:_process_minibatch] Loss is 0.47116899490356445
+[2020-04-14 02:46:34,037] [INFO] [worker.py:879:_process_minibatch] Loss is 0.9414381384849548
 ```
+
+To see the evaluation metrics in master pod:
```
Comment (skydoorkai) [inline docs/tutorials/elasticdl_local.md]: To see evaluation metrics in the master pod:

File: docs/tutorials/elasticdl_local.md (hunk 1 above)
Comment (QiJune) [reply docs/tutorials/elasticdl_local.md]: Done

File: docs/tutorials/elasticdl_local.md (hunk 2 above)
Comment (QiJune) [reply docs/tutorials/elasticdl_local.md]: Done

File: docs/tutorials/elasticdl_local.md (hunk 3 above)
Comment (QiJune) [reply docs/tutorials/elasticdl_local.md]: Done

File: docs/tutorials/elasticdl_local.md (hunk 4 above)
Comment (QiJune) [reply docs/tutorials/elasticdl_local.md]: Done

File: docs/tutorials/elasticdl_local.md [hunk 5]
```diff
@@ -12,37 +12,45 @@ In this tutorial, we use [hyperkit](https://github.com/moby/hyperkit) as the hyp
...
+pip install -r elasticdl/requirements.txt
 python setup.py install
 ```
 
-### Setup kubernetes related environment
+### Setup Kubernetes related environment
 
 ```bash
-minikube start --vm-driver=hyperkit --cpus 2 --memory 6144
+minikube start --vm-driver=hyperkit --cpus 2 --memory 6144 --disk-size=50gb
```
Comment (workingloong) [inline docs/tutorials/elasticdl_local.md]: 50gb maybe huge?

File: docs/tutorials/elasticdl_local.md [hunk 6]
```diff
@@ -54,66 +62,101 @@ elasticdl train \
...
   --job_name=test-mnist \
   --log_level=INFO \
   --image_pull_policy=Never \
-  --output=model_output
+  --distribution_strategy=ParameterServerStrategy
 ```
 
-We can also use the pre-built docker image to run the ElasticDL job.
-In the following command, the pre-built image name is `elasticdl:mnist`, the absolute model zoo path inside the docker is `/model_zoo`.
+`image_base` is the base docke image argument. A new image will be built based on it each time while submitting the Elastic job.
```
Comment (workingloong) [inline docs/tutorials/elasticdl_local.md]: "base docke" -> "base docker"

File: docs/tutorials/elasticdl_local.md (hunk 5 above)
Comment (QiJune) [reply docs/tutorials/elasticdl_local.md]: Done. change to 20gb

File: docs/tutorials/elasticdl_local.md (hunk 6 above)
Comment (QiJune) [reply docs/tutorials/elasticdl_local.md]: Done

**Review by skydoorkai — APPROVED** [review]

**Pull request merged**